modules/atomic-widgets/styles/utils.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\Styles;
+
+class Utils {
+	public static function generate_id( string $prefix = '', $existing_ids = [] ): string {
+		$existing_ids = array_flip( $existing_ids );
```

Comment:
Why there is a need for flip? do you pass an array or ids? `['id-1', 'id-2', 'id-3']` if yes (it seems like you pass it the right way) there is no need to flip